lib/presentation/widgets/m3_widgets/text_fields/trips_text_field.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:trip_tally/presentation/theme/app_dimensions.dart';
+import 'package:trip_tally/presentation/utils/enums/context_extensions.dart';
+
+class TripsTextField extends StatelessWidget {
```

Comment:
This file should be in the `trips_history_page` folder as it is associated with the history of the trip. `TripsTextField` doesn't says much, I would rather name it `TripsHistorySearchBar` or `TripsHistorySearchTextField`